packages/features/bookings/lib/getAllCredentialsForUsersOnEvent/getAllCredentials.ts

```diff
@@ -72,5 +67,45 @@ export const getAllCredentials = async (
     }
   }
 
+  // Only return CRM credentials that are enabled on the event type
+  const eventTypeAppMetadata = eventType?.metadata?.apps;
+
+  // Will be [credentialId]: { enabled: boolean }]
+  const eventTypeCrmCredentials = {};
+
+  for (const appKey in eventTypeAppMetadata) {
+    const app = eventTypeAppMetadata[appKey];
+    if (app.appCategories && app.appCategories.some((category) => category === "crm")) {
+      // eventTypeCRMCredentials.push({ credentialId: app.credentialId, enabled: app.enabled });
+      eventTypeCrmCredentials[app.credentialId] = {
+        enabled: app.enabled,
+      };
+    }
+  }
+
+  allCredentials = allCredentials.filter((credential) => {
+    if (!credential.type.includes("_crm") && !credential.type.includes("_other_calendar")) {
+      return credential;
+    }
+
+    // Backwards compatibility: All CRM apps are triggered for every event type. Unless disabled on the event type
+    // Check if the CRM app exists on the event type
+    if (eventTypeCrmCredentials[credential.id]) {
+      if (eventTypeCrmCredentials[credential.id].enabled) {
+        return credential;
+      }
+    } else {
```

Comment:
If the CRM credential exists in the event type metadata and is enabled, then return the credential